lib/auth/apiserver.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2015 Gravitational, Inc.
+Copyright 2017 Gravitational, Inc.
```

Comment:
Updating the copyright year is not necessary. In fact, IIRC it's actually not right. The year is originally supposed to mean "year of first time publication". So it's either 2015 or 2015-17, but not 2017.